routes/index.js

```diff
@@ -1,13 +1,52 @@
 const express = require("express");
 const router = express.Router();
 const Post = require("../models/post");
+var cors = require('cors')
+var app = express();
+var multer  = require('multer')
+var upload = multer({ dest: './public/images' })
+// app.use(express.static("public"))
+
+app.use(cors())
+
+// router.post("/upload",  upload.single('file'),function(req,res,next) {
+//     console.log("author :",req.body.author)
+//     console.log("location ",req.body.location)
+//     console.log("file:" ,req.file)
+    
+    
+// })
+
+router.post("/upload",  upload.single('file'),function(req,res,next) {
+    var data = {
+        author : req.body.author,
+        location : req.body.location,
+        filePath: req.file.path,
```

Comment:
have you checked if you getting the correct image path ??

Reply:
yes, It's getting the correct path sir